src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.stars-star {
+
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-size: center;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars-star--active {
+  background-image: url(images/star-active.svg);
+  background-size: center;
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Use block modifier to get the active stars. Also, you can change only the background-image.
All other properties are the same
![image](https://github.com/mate-academy/layout_stars/assets/96409563/2bd414ca-960f-4ae5-8ea5-5a53ef43dbb4)